openshift/kebechet-template.yaml

```diff
@@ -0,0 +1,102 @@
+# This template uses the kebechet-job argo workflow to run kebechet on a particular webhook. 
+apiVersion: v1
+kind: Template
+metadata:
+  name: kebechet-job
```

Comment:
Just kebechet? I think there will be no "job". Here and at other places as well.

Reply:
I will change it :)